Dockerfile

```diff
@@ -67,7 +67,7 @@ ENV CATALINA_HOME /usr/local/tomcat
 ENV CATALINA_BASE /usr/local/tomcat
 ENV CATALINA_TMPDIR /usr/local/tomcat/temp
 ENV PATH $CATALINA_HOME/bin:$PATH
-ENV JRE_HOME /usr
+# ENV JRE_HOME /usr
```

Comment:
Maybe leave out since Git history has it?

Reply:
yeah, forgot to delete it.